tox.ini

```diff
@@ -62,7 +62,7 @@ basepython = python3.11
 deps =
     grpc-stubs
     mypy
-commands = mypy lekko_client --install-types --non-interactive --strict
+commands = mypy lekko_client --install-types --non-interactive --strict --warn-unreachable
```

Comment:
Some of the code that I changed was falsely considered unreachable to mypy and was silently not type-checked. This flag makes mypy to complain about unreachable code. It turned out that we had only one instance of it, so I enabled the warning and fixed.